lib/krane/kubernetes_resource/pod.rb

```diff
@@ -10,6 +10,7 @@ class Pod < KubernetesResource
     )
 
     attr_accessor :stream_logs
+    attr_reader :definition
```

Comment:
Allows StatefulSet resource to access each Pod's `definition["metadata"]["labels"]["controller-revision-hash"]` value, a new addition to how we check for successful deploys